### PR TITLE
[indicators] Add new port

### PR DESCRIPTION
**Pull request by myd7349**
https://github.com/p-ranav/indicators

### CODE REVIEW

**Comment by cenit** [comment]
Why are you patching the cmakelists to avoid installing headers in a indica subfolder?

**Comment by myd7349** [comment]
> 
> 
> Why are you patching the cmakelists to avoid installing headers in a indica subfolder?

`${CMAKE_CURRENT_LIST_DIR}/include/indica.hpp` doesn't exist anymore. Now the headers are in `include/indicators`: https://github.com/p-ranav/indicators/tree/master/include/indicators.

**Comment by myd7349** [comment]
https://github.com/p-ranav/indicators/pull/6

File: ports/indicators/CONTROL [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Source: indicators
+Version: 2019-12-17
```
Comment (NancyLi1013) [inline ports/indicators/CONTROL]: Why not use a release version here? I noticed there was a release version v1.1.

File: ports/indicators/portfile.cmake [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# header-only library
+
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO p-ranav/indicators
+    REF 24d92beacafd1348d48eef6d24c6d949b4487a33
```
Comment (NancyLi1013) [inline ports/indicators/portfile.cmake]: The same here. We should use commit id 2edbccce9c24e92439d11c19b592e82371c2b477 that has released.

File: ports/indicators/portfile.cmake [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+)
+
+vcpkg_install_cmake()
+
+vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/indica TARGET_PATH share/indica)
+
+file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug ${CURRENT_PACKAGES_DIR}/lib)
+
+file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
+file(INSTALL ${SOURCE_PATH}/LICENSE.termcolor DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT})
```
Comment (NancyLi1013) [inline ports/indicators/portfile.cmake]: Please remove the last line. We need one license is enough.

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

**Review by NancyLi1013 — APPROVED** [review]

**Pull request merged**